sns/cm/repair/ut/cm.c

```diff
@@ -43,6 +43,7 @@
 #include "mdservice/md_fid.h"
 #include "rm/rm_service.h"                 /* m0_rms_type */
 #include "sns/cm/repair/ag.h"
+#include "sns/cm/rebalance/ag.c"
```

Comment:
Why do we need to include this? Export the required declarations through the .h file and include that instead.

Reply:
Thanks Mehul this import was not needed. I have fixed this in the latest commit.